### DISCUSSION TITLE
How to fix top and bottom plates

### DISCUSSION THREAD

**Discussion by Ali141272** [neperfepx/neper#464] [Q&A] [answered]
I have a query about how to fix the top and bottom plates in Neper. I studied the documentation of Neper, but I am a new user and didn't find the commands, through which I can generate the top and bottom plates.
I need your kind suggestions, on how to build these plates in Neper.
Thank you very much.
Regards,
Ali
![222](https://user-images.githubusercontent.com/102291665/166723318-8bbef999-25c6-48e9-b6d3-6a70979019dd.png)

**Comment by mkasemer** [comment, marked as answer]
What do you mean by "fix", and "plates"? Do you mean apply boundary conditions for your simulations such that the top and bottom surfaces do not move?

If this is the case, then I do not think that this is a question for Neper, but rather for whatever software you are using for simulations.

**Reply by Ali141272** [reply]
Sir thank you for your timely response, please have a look at the attachment, I built these top and bottom plates in Gmsh, and now my main query is, how to build these plates in Neper.
![Top   Bottom plates](https://user-images.githubusercontent.com/102291665/166799774-70cb0d77-83a8-4dd6-a4ac-699db20966fb.png)

**Reply by mkasemer** [reply]
While I am unsure if Neper can generate these (I assume not), I wonder why you need them? Are you performing deformation simulations? Are the plates rigid, compared to the polycrystal? If so, then simply choose boundary conditions on the nodes of the top and bottom surfaces that would mimic the effects of a rigid plate. In this case, it is unnecessary to explicitly include them.

**Reply by Ali141272** [reply]
Sir thanks once again for your positive response, I will try my best.
